atom/common/api/atom_api_crash_reporter.cc

```diff
@@ -31,17 +33,39 @@ struct Converter<std::map<std::string, std::string> > {
   }
 };
 
+template<>
+struct Converter<std::vector<CrashReporter::UploadReportResult> > {
```

Comment:
It is OK to use `>>` in template now: https://chromium-cpp.appspot.com.

And we only need to define converter for `CrashReporter::UploadReportResult` here, there is already a generic converter for `std::vector<*>`.